corehq/apps/reports/standard/forms/reports.py

```diff
@@ -131,12 +131,22 @@ def _fmt_date(somedate):
             else:
                 form_name = EMPTY_FORM
                 form_username = EMPTY_USER
+
+            if xform_dict['doc_type'] == "XFormArchived":
+                archive_dates = [operation['date'] for operation in xform_dict['history']
+                                 if operation['operation'] == 'archive']
```

Comment:
would have got bonus points if you pulled out the magic string `'archive'` into a constant.

Reply:
what's the best way of doing this? 
both couch: 
https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/couchforms/models.py#L327-L330
and SQL: 
https://github.com/dimagi/commcare-hq/blob/master/corehq/form_processor/models.py#L543
define these in different places.
Do you think I should pull this out in into a constant somewhere else?

Reply:
yuck. no strong opinions. i guess you could do that or you could define a couch-only constant or you could leave it.